config/install/node.type.planets.yml

```diff
@@ -0,0 +1,16 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - menu_ui
+third_party_settings:
+  menu_ui:
+    available_menus: {  }
+    parent: ''
+name: Planets
+type: planets
+description: 'A list of Planet Resources, using data from The Star Wars API(SWAPI)'
```

Comment:
Add a period at the end of the description to match others.

Reply:
Done